LTI-SSF/prompts.md

```
@@ -0,0 +1,169 @@
+
+He generado una carpeta con:
+
+* `LTI-XXX.md` (sustituye *XXX* por tus iniciales)
+* `prompts.md` (prompts prácticos para cada fase)
+
+Descárgalo aquí:
+
+* [Descargar carpeta comprimida](sandbox:/mnt/data/LTI-XXX.zip)
+* O abre individualmente:
+
+    * [LTI-XXX.md](sandbox:/mnt/data/LTI-XXX/LTI-XXX.md)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)